CHANGES.md

```diff
@@ -0,0 +1,7 @@
+# Change Log / Release Notes
+
+## unknown
+
+## 0.3.1 (2021-05-09)
```

Comment:
0.3.1 was released in 2011

Reply:
Oh, indeed, I was thinking that 0f0066a already made it to Hackage as 0.3.1.  Should've read more carefully.  In that case the version number should stay at 0.3.3 as well, I suppose